paimon-core/src/main/java/org/apache/paimon/manifest/SimpleManifestEntry.java

```diff
@@ -0,0 +1,123 @@
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.manifest;
+
+import org.apache.paimon.data.BinaryRow;
+import org.apache.paimon.types.DataField;
+import org.apache.paimon.types.IntType;
+import org.apache.paimon.types.RowType;
+import org.apache.paimon.types.TinyIntType;
+import org.apache.paimon.utils.Preconditions;
+
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Objects;
+
+import static org.apache.paimon.utils.SerializationUtils.newBytesType;
+import static org.apache.paimon.utils.SerializationUtils.newStringType;
+
+/** Partition Entry pojo read from manifest file. */
+public class SimpleManifestEntry {
+    private final FileKind kind;
+
+    private final BinaryRow partition;
+    private final int bucket;
+
+    private final String fileName;
+
+    public FileKind kind() {
+        return kind;
+    }
+
+    public BinaryRow partition() {
+        return partition;
+    }
+
+    public int bucket() {
+        return bucket;
+    }
+
+    public String getFileName() {
+        return fileName;
+    }
+
+    public SimpleManifestEntry(FileKind kind, BinaryRow partition, int bucket, String fileName) {
+        this.kind = kind;
+        this.partition = partition;
+        this.bucket = bucket;
+        this.fileName = fileName;
+    }
+
+    @Override
+    public boolean equals(Object obj) {
+        if (!(obj instanceof SimpleManifestEntry)) {
+            return false;
+        }
+        SimpleManifestEntry eobj = (SimpleManifestEntry) obj;
+
+        return Objects.equals(eobj.fileName, this.fileName)
+                && Objects.equals(eobj.partition, this.partition)
+                && Objects.equals(eobj.kind, this.kind);
+    }
+
+    public static RowType schema() {
+        List<DataField> fields = new ArrayList<>();
+        fields.add(new DataField(0, "_KIND", new TinyIntType(false)));
+        fields.add(new DataField(1, "_PARTITION", newBytesType(false)));
+        fields.add(new DataField(2, "_BUCKET", new IntType(false)));
+        fields.add(new DataField(3, "_FILE", schemaDataFileName()));
+        return new RowType(fields);
+    }
+
+    public static RowType schemaDataFileName() {
+        List<DataField> fields = new ArrayList<>();
+        fields.add(new DataField(0, "_FILE_NAME", newStringType(false)));
+        return new RowType(fields);
+    }
+
+    public static List<SimpleManifestEntry> mergePartitionEntries(
```

Comment:
Can we unify this to `ManifestEntry.mergeEntries`?
Maybe it can be:
```
ManifestEntry {
    public static <T> Collection<T> mergeEntries(
            List<T> entries, Function<T, Identifier> extractIdentifier) {
            ..............
    }
}
```